github/ci/testgrid/gen-config.yaml

```diff
@@ -1,9 +1,11 @@
 dashboards:
 - name: kubevirt-presubmits
 - name: kubevirt-periodics
+- name: kubevirt-libguestfs-appliance-periodics
```

Comment:
```suggestion
- name: libguestfs-appliance-periodics
```

Reply:
I had an issue with that, something about needing a proper prefix which libguestfs doesn't fit, see this [comment](https://github.com/kubevirt/project-infra/pull/1267#issuecomment-852406685)

Reply:
Sorry, didn't see your previous comment :) And didn't know about this requirement, then probably we need to change the name of the existing dashboards to something like `kubevirt-kubevirt-periodics`. In any case the changes in this PR are ok, thanks!